docs/cmdline-opts/ipfs-gateway.d

```diff
@@ -0,0 +1,36 @@
+c: Copyright (C) 2022, Mark Gaiser, <mark.gaiser@protocol.ai>
+SPDX-License-Identifier: curl
+Long: ipfs-gateway
+Arg: <URL>
+Help: Defines the gateway to handle IPFS protocol requests.
+Added: 7.87.0
+See-also: help manual
+Category: ipfs
+Example: --ipfs-gateway $URL ipfs://
```

Comment:
This is not a complete example. Even if we want to keep it short, shouldn't the `ipfs://` part at least get a few hash-looking letters to the right of the double-slashes?

Reply:
I'll add the big buck bunny hash as an example. It's open source, non affiliated and should even work when a user wants to try it out.